homeassistant/components/somfy/cover.py

```diff
@@ -0,0 +1,106 @@
+"""
+Support for Somfy Covers.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/cover.somfy/
+"""
+
+from homeassistant.components.cover import CoverDevice, ATTR_POSITION, \
+    ATTR_TILT_POSITION
+from homeassistant.components.somfy import DOMAIN, SomfyEntity, DEVICES, API
+
+DEPENDENCIES = ['somfy']
```

Comment:
This can be removed as a result of adding the manifest.json file.

Reply:
Add a placeholder `async_setup_platform` coroutine that just passes. It's currently required and expected by our platform helper.

Reply:
```suggestion

async def async_setup_platform(
        hass, config, async_add_entities, discovery_info=None):
    """Old way of setting up platform.
    Can only be called when a user accidentally mentions the platform in their
    config. But even in that case it would have been ignored.
    """
    pass
```